landlab/components/landslides/landslide_probability.py

```diff
@@ -441,32 +526,72 @@ def __init__(
             ), "Input array should be of the length of grid.number_of_nodes!"
             self._recharge_mean = groundwater__recharge_mean
             self._recharge_stdev = groundwater__recharge_standard_deviation
-        # Custom HSD inputs - Hydrologic Source Domain -> Model Domain
+        # Recharge Custom HSD inputs - Hydrologic Source Domain -> Model Domain
         elif self._groundwater__recharge_distribution == "data_driven_spatial":
             self._HSD_dict = groundwater__recharge_HSD_inputs[0]
             self._HSD_id_dict = groundwater__recharge_HSD_inputs[1]
             self._fract_dict = groundwater__recharge_HSD_inputs[2]
             self._interpolate_HSD_dict()
-
-        # Check if all output fields are initialized
+  
+        # Following code will deal with the input distribution and associated
+        # parameters for depth to water table hydrologic forcing
+        # Depth to water table - Uniform distribution
+        if self._groundwater__depth_distribution == "uniform":
+            self._depth_min = groundwater__depth_min_value
+            self._depth_max = groundwater__depth_max_value
+            self._De = np.random.uniform(self._depth_min, self._depth_max, size=self._n)
+         # Depth to water table - Lognormal Distribution - Uniform in space
+        elif self._groundwater__depth_distribution == "lognormal":
+            assert (
+                groundwater__depth_mean is not None
+            ), "Input mean of the distribution!"
+            assert (
+                groundwater__depth_standard_deviation is not None
+            ), "Input standard deviation of the distribution!"
+            self._depth_mean = groundwater__depth_mean
+            self._depth_stdev = groundwater__depth_standard_deviation
+            self._mu_lognormal = np.log(
+                (self._depth_mean ** 2)
+                / np.sqrt(self._depth_stdev ** 2 + self._depth_mean ** 2)
+            )
+            self._sigma_lognormal = np.sqrt(
+                np.log((self._depth_stdev ** 2) / (self._depth_mean ** 2) + 1)
+            )
+            self._De = np.random.lognormal(
+                self._mu_lognormal, self._sigma_lognormal, self._n
+            )    
+    
+    # Depth to water table - Lognormal Distribution - Variable in space                                  
+        elif self._groundwater__depth_distribution == "lognormal_spatial":
+<<<<<<< HEAD
+=======
```

Comment:
@ChristinaB if you look at the errors in the build logs (for example, [this one](https://travis-ci.org/github/landlab/landlab/jobs/687348323#L975) you can see that part of the problem is that you have unresolved merge commits, for example here. 

Once this sort of thing is resolved, there may be other things that keep the tests from passing, but this is where it is getting hung up now.

Reply:
Thanks!  I saw that in the log and wasn't sure how it got added or how to avoid it in the future. But I will go in and try to fix it and keep trying until it passes!

Reply:
@ChristinaB the <<<<<<<HEAD text is the telltale sign of an unresolved merge conflict... 

Are you able to run the tests locally with pytest?

Usually you can run your components tests only (as opposed to the entire repository) with 

```
pytest */componet/name_of_component
```

this tests the files in landlab and in tests. 

Since you have tests that are currently in notebooks, you'll need to run 

```
pytest --run-notebook
```

since the default behavior of pytest has been configured to ignore the notebooks.

Reply:
These all passed on my dev. environment. Finally.